config/install/core.entity_view_display.node.stanford_page.default.yml

```diff
@@ -0,0 +1,103 @@
+uuid: 219690b7-c9eb-42a7-b7fd-a2bad54d1079
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.stanford_page.su_page_components
+    - node.type.stanford_page
+  module:
+    - entity_reference_revisions
+    - jumpstart_ui
+    - layout_builder
+    - layout_discovery
+    - user
+third_party_settings:
+  layout_builder:
+    allow_custom: false
+    enabled: true
+    sections:
+      -
+        layout_id: layout_onecol
```

Comment:
Do we need this section? If so, can we switch to the `basic` layout in jumpstart_ui so we match the responsive grid settings?

Reply:
oh i think this just needs to be removed. i'm using the pacman_left layout below this line. which doesnt work as a dynamic layout. the left sidebar is always visible even if no content exists. But we can visit that later.